docs/1.getting-started/4.styling.md

```diff
@@ -0,0 +1,440 @@
+</template>
+
+<style module>
+.red {
+  color: red;
+}
+</style>
+```
+
+### Preprocessor support
+
+SFC style blocks support pre-processors syntax. Vite come with built-in support for .scss, .sass, .less, .styl and .stylus files without configuration. You just need to install them first, and they will be available directly in SFC with the lang attribute.
+
+::code-group
+
+```html [SCSS]
+<style lang="scss">
+  /* Write scss here */
+</style>
+```
+
+```html [SASS]
+<style lang="sass">
+  /* Write sass here */
+</style>
+```
+
+```html [LESS]
+<style lang="less">
+  /* Write less here */
+</style>
+```
+
+```html [STYLUS]
+<style lang="stylus">
+  /* Write stylus here */
+</style>
+```
+
+::
+
+You can refere to the [vite css docs](https://vitejs.dev/guide/features.html#css) and the [@vitejs/plugin-vue docs](https://github.com/vitejs/vite-plugin-vue/tree/main/packages/plugin-vue).
+For webpack users, refer to the [vue loader docs](https://vue-loader.vuejs.org/).
+
+## Using PostCSS
+
+Nuxt comes with postcss built-in. You can configure it in your `nuxt.config` file.
+
+```ts [nxut.config.ts]
+export default defineNuxtConfig({
+    postcss: {
+      plugins: {
+        'postcss-nested': {}
+        "postcss-custom-media": {}
+      }
+    }
+})
+```
+
+For proper syntax highlighting in SFC, you can use the postcss lang attribute.
+
+```html
+<style lang="postcss">
+  /* Write stylus here */
+</style>
+```
+
+By default, Nuxt comes with the following plugins already pre-configured :
+
+- [postcss-import](https://github.com/postcss/postcss-import): Improves the `@import` rule
+- [postcss-url](https://github.com/postcss/postcss-url) : Transform `url()` statements
+- [autoprefixer](https://github.com/postcss/autoprefixer) : Automatically add vendor prefixes
+- [cssnano](https://cssnano.co/) : Minification and purge
+
+##  Leveraging Layouts for Multiple Styles
+
+If you need to style different parts of your application completely differently, you can use layouts.
+Use different styles for different layouts.
+
+```html
+<template>
+  <h1> Default Layout</h1>
+  <slot />
+</template>
+<style>
+.DefaultLayout {
+  color: red;
+}
+</style>
+```
+
+::ReadMore{link="docs/guide/directory-structure/layouts"}
+::
+
+## Third party libraries and modules
+
+Nuxt isn't opinionated when it comes to styling and provides you with a wide variety of options. You can use any styling tool that you want, such as the popular [UnoCSS](https://unocss.dev/) or [Tailwind CSS](https://tailwindcss.com/).
+
+The community and the Nuxt team have developped plenty of Nuxt modules to makes the integration easier.
+Here's a non exhaustive list of modules that you can check-out to get started :
+
+- [Uno CSS](https://nuxt.com/modules/unocss)
+- [Tailwind CSS](https://nuxt.com/modules/tailwindcss)
+- [Fontaine](https://github.com/nuxt-modules/fontaine)
+- [Pinceau](https://pinceau.dev/)
+- [NuxtLabs UI](https://nuxt.com/modules/nuxtlabs-ui)
+
```

Comment:
I think it would be great to emphasize more the Nuxt modules link with a filter preset for styling options

Reply:
Yes we could do that ! However I cherry picked this 5 because they are maintained by members of the Nuxt team if I'm not mistaken (danielroe, Atinux, antfu and Tahul).
However these 5 are not listed under the same section or even not listed, so I still think we should recommend them.

Reply:
For sure 👍 Just adding a link would be enough like `[Explore more modules from the community](...)`